### PR TITLE
Don't default to starting Necrozma-Ultra in its base forme

### PR DESCRIPTION
**Pull request by MacChaeger**

### CODE REVIEW

File: data/rulesets.js [hunk 1]
```diff
@@ -215,13 +215,17 @@ exports.BattleFormats = {
...
+						problems.push("Necrozma-Ultra must start the battle as Necrozma-Dawn-Wings or Necrozma-Dusk-Mane holding Ultranecrozium Z."); // Necrozma-Ultra transforms from one of two formes, and neither one is the base forme
+					} else if (!template.requiredItems.includes(item.name)) {
+						problems.push("" + template.species + " transforms in-battle with " + Chat.plural(template.requiredItems.length, "either ") + template.requiredItems.join(" or ") + '.'); // Mega or Primal
+					}
 				}
 				if (template.requiredMove && set.moves.indexOf(toId(template.requiredMove)) < 0) {
 					problems.push("" + template.species + " transforms in-battle with " + template.requiredMove + "."); // Meloetta-Pirouette, Rayquaza-Mega
 				}
-				if (!format.noChangeForme) set.species = template.baseSpecies; // Fix forme for Aegislash, Castform, etc.
+				if (!format.noChangeForme) set.species = template.baseSpecies; // Fix forme
```
Comment (Zarel) [inline data/rulesets.js]: Why??????

File: data/rulesets.js [hunk 2]
```diff
@@ -215,13 +215,17 @@ exports.BattleFormats = {
 				if (template.requiredAbility && set.ability !== template.requiredAbility) {
 					problems.push("" + template.species + " transforms in-battle with " + template.requiredAbility + "."); // Darmanitan-Zen, Zygarde-Complete
 				}
-				if (template.requiredItems && !template.requiredItems.includes(item.name)) {
-					problems.push("" + template.species + " transforms in-battle with " + Chat.plural(template.requiredItems.length, "either ") + template.requiredItems.join(" or ") + '.'); // Mega, Primal, or Ultra
+				if (template.requiredItems) {
+					if (template.species === 'Necrozma-Ultra') {
+						problems.push("Necrozma-Ultra must start the battle as Necrozma-Dawn-Wings or Necrozma-Dusk-Mane holding Ultranecrozium Z."); // Necrozma-Ultra transforms from one of two formes, and neither one is the base forme
+					} else if (!template.requiredItems.includes(item.name)) {
+						problems.push("" + template.species + " transforms in-battle with " + Chat.plural(template.requiredItems.length, "either ") + template.requiredItems.join(" or ") + '.'); // Mega or Primal
```
Comment (Zarel) [inline data/rulesets.js]: Template string this...

File: data/rulesets.js (hunk 1 above)
Comment (MacChaeger) [reply data/rulesets.js]: That was there because Aegislash, Castform, etc. didn't have `requiredAbility`, wasn't it?

File: data/rulesets.js (hunk 2 above)
Comment (MacChaeger) [reply data/rulesets.js]: OK I'll just template string all of them then.

File: data/rulesets.js (hunk 1 above)
Comment (Zarel) [reply data/rulesets.js]: Oh, I see. Should probably clarify then "fix forme if no requiredAbility/requiredMove"

Comment (MacChaeger) [reply data/rulesets.js]: Well, it actually sets the forme for every battleForme, even the ones that do have something required. How about `// Fix battle-only forme`?

Comment (Zarel) [reply data/rulesets.js]: Sure.

**Pull request merged**